### PR TITLE
feat: make studio UI aware of override

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Make ORA studio UI aware of flexible course-wide override

JIRA:  https://2u-internal.atlassian.net/browse/AU-1322

**What changed?**

- Add override to studio context
- If it's enabled, disable the select box for flexbile grading
- If it's enabled, show a message that the setting is enabled.

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/openedx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in [openassessment/\_\_init\_\_.py](https://github.com/openedx/edx-ora2/blob/master/openassessment/__init__.py#L4) and [package.json](https://github.com/openedx/edx-ora2/blob/master/package.json#L3)

**Testing Instructions**

- Navigate to an ORA with a peer step in a course with the override turned off. The flexible setting should be active and there should be no message about the override.
- Turn the override on
- The input in the ORA settings should now be disabled and there should be a warning message.

Screenshot 2023-07-14 at 11 34 31 AM

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @openedx/content-aurora

### CODE REVIEW

**Review by leangseu-edx — APPROVED** [review]
LGTM 👍
